components/collapsible/basic/src/index.scss

```diff
@@ -2,15 +2,31 @@
 @import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
 @import '~@schibstedspain/sui-theme/lib/index';
 
+$p-collapsible-basic-trigger: 0 !default;
```

Comment:
I understood these variables should be placed in an external file named `_settings.scss`, isn't it?

Reply:
didn't know it. Done!